From 4f0b91f038c4dba1ed17809e70bf83a575a391c7 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Wed, 9 Aug 2017 13:01:35 +0800
Subject: [PATCH 2359/5242] MLK-16168 gpu: Move imx up before drm in Makefile

commit  262fc4152da16cec52fdc1d9d1ef5461e0a77d73 from
https://source.codeaurora.org/external/imx/linux-imx.git

We register dpu and ipu-v3 crtc platform devices in dpu and ipu-v3
base drivers for imx-drm to use, so it would be straight forward to
put imx before drm in Makefile.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/Makefile b/drivers/gpu/Makefile
index 368f426..beaa25a 100644
--- a/drivers/gpu/Makefile
+++ b/drivers/gpu/Makefile
@@ -2,5 +2,5 @@
 # taken to initialize them in the correct order. Link order is the only way
 # to ensure this currently.
 obj-$(CONFIG_TEGRA_HOST1X)	+= host1x/
-obj-y			+= drm/ vga/
 obj-y			+= imx/
+obj-y			+= drm/ vga/
-- 
1.7.9.5

